src/scripts.js

```diff
@@ -291,57 +242,52 @@ function generateUser(userInfo) {
 //   hideUnselectedRecipes(found);
 // }
 
-// function createRecipeObject(recipes) {
-//   recipes = recipes.map(recipe => new Recipe(recipe));
-//   return recipes
-// }
 
-// function toggleMenu() {
-//   var menuDropdown = document.querySelector(".drop-menu");
-//   menuOpen = !menuOpen;
-//   if (menuOpen) {
-//     menuDropdown.style.display = "block";
-//   } else {
-//     menuDropdown.style.display = "none";
-//   }
-// }
 
-// function showAllRecipes() {
-//   recipes.forEach(recipe => {
-//     let domRecipe = document.getElementById(`${recipe.id}`);
-//     domRecipe.style.display = "block";
-//   });
-//   showWelcomeBanner();
-// }
+
+
 
 // // CREATE AND USE PANTRY
-// function findPantryInfo() {
-//   user.pantry.forEach(item => {
-//     let itemInfo = ingredientsData.find(ingredient => {
-//       return ingredient.id === item.ingredient;
-//     });
-//     let originalIngredient = pantryInfo.find(ingredient => {
-//       if (itemInfo) {
-//         return ingredient.name === itemInfo.name;
-//       }
-//     });
-//     if (itemInfo && originalIngredient) {
-//       originalIngredient.count += item.amount;
-//     } else if (itemInfo) {
-//       pantryInfo.push({name: itemInfo.name, count: item.amount});
-//     }
-//   });
-//   displayPantryInfo(pantryInfo.sort((a, b) => a.name.localeCompare(b.name)));
-// }
+function findPantryInfo(ingredientsRepository) {
+  const fullPantryIngredients = []
+  const pantryIngredients = user.pantry.ingredients
+
+  pantryIngredients.forEach(pantryIngredient => {
+    fullPantryIngredients.push(ingredientsRepository.getIngredientName(pantryIngredient))
+  })
+  // console.log('fullPantry', fullPantryIngredients);
+  // console.log('pantry', pantryIngredients);
+  // console.log(pantryIngredients);
+  // user.pantry.ingredients.forEach(item => {
+  //   let itemInfo = ingredientsData.find(ingredient => {
+  //     return ingredient.id === item.ingredient;
+  //   });
+  //   let originalIngredient = user.pantry.ingredients.find(ingredient => {
+  //     if (itemInfo) {
+  //       return ingredient.name === itemInfo.name;
+  //     }
+  //   });
+  //   if (itemInfo && originalIngredient) {
+  //     originalIngredient.count += item.amount;
+  //   } else if (itemInfo) {
+  //     user.pantry.ingredients.push({name: itemInfo.name, count: item.amount});
+  //   }
+  // });
+  // // displayPantryInfo(user.pantry.ingredients.sort((a, b) => a.name.localeCompare(b.name)));
+  // console.log(user.pantry);
+  // console.log(fullPantryIngredients[0]);
```

Comment:
I can't quite remember what it is this commented-out portion does, we should figure that out and delete it if it's superfluous!